stable/grafana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 1.1.0
+version: 1.1.1
```

Comment:
Please increment the new chart version to be greater than the existing version of 1.1.1